safenode/src/network_transfers/mod.rs

```diff
@@ -49,12 +49,19 @@ impl Transfers {
     }
 
     /// Get the required fee for the specified spend priority.
-    pub(crate) fn get_required_fee(&self, dbc_id: DbcId, priority: SpendPriority) -> RequiredFee {
+    pub(crate) fn get_required_fee(
+        &self,
+        dbc_id: DbcId,
+        priority: SpendPriority,
+    ) -> (NodeId, RequiredFee) {
```

Comment:
this then means the `RequiredFee` is charged by this node, right?

but our `node_reward_key` is already included within the `RequiredFee`, which can be accessed

Reply:
> this then means the RequiredFee is charged by this node, right?
Yes.

> but our node_reward_key is already included within the RequiredFee, which can be accessed
We need the node id when the fee ciphers are sent in to the nodes. This is how each node can find their set of fee ciphers within a spend request.